source/deploy/quick-start-evaluation.rst

```diff
@@ -0,0 +1,72 @@
+Quick Start Evaluation
+======================
+
+This guide provides instructions for quickly trying out Mattermost using either Docker or Azure Marketplace. These options are ideal for testing and evaluation purposes as they allow you to quickly get a Mattermost instance up and running for exploration and testing. However, these quick start options are not recommended for production use. They use SQLite as the database and are configured for demonstration purposes only.
+
+Deployment options
+------------------
+
+.. tab:: Azure Marketplace
+
+    Mattermost is available as a pre-configured virtual machine image in the Azure Marketplace. This option is preferred for customers already using Azure, as it integrates seamlessly within their existing Azure infrastructure.
+
+    **Prerequisites:**
+
+    * An Azure subscription
+    * Basic familiarity with Azure Portal
+
+    **Steps to deploy Mattermost on Azure:**
+
+    1. Visit the `Mattermost, Collaboration for Mission-Critical Work (VM) <https://azuremarketplace.microsoft.com/de-de/marketplace/apps/mattermost.mattermost-all-in-one?tab=overview>`_
+
+    2. Select **Get it now** and sign in to your Azure account.
+
+    3. Configure the deployment:
+        - Choose your subscription
+        - Create or select a resource group
+        - Select a region
+        - Choose a VM size (recommended: at least 2 vCPUs, 4GB RAM)
+        - Configure network settings
+        - Set up your admin credentials
+
+    4. Review and create the deployment.
+
+    5. Once deployed, access your Mattermost instance using the public IP address or DNS name provided.
+
+    .. note::
+        The Azure Marketplace image comes with PostgreSQL and is more suitable for testing production-like scenarios. Remember to delete the resources when you're done to avoid unnecessary charges.
```

Comment:
In the doc intro blurb it states that both these deploy options use SQLite databases so are not configured for production. It this statement here contradictory? @DSchalla

Reply:
@nickmisasi @toninis Can you help here as I don't have much background with the AIO image?

Reply:
Perfect, thanks Nick. Let's update the wording and then good to merge.